migrator.go

```diff
@@ -108,7 +96,18 @@ func (m spannerMigrator) CreateTable(values ...interface{}) error {
 				values                  = []interface{}{m.CurrentTable(stmt)}
 				hasPrimaryKeyInDataType bool
 			)
-
+			for _, f := range stmt.Schema.Fields {
+				// Cloud spanner does not support auto incrementing primary keys.
+				if f.AutoIncrement && f.HasDefaultValue && f.DefaultValue == "" && f.DefaultValueInterface == nil {
+					if err := tx.Exec("CREATE SEQUENCE IF NOT EXISTS " +
+						stmt.Table + "_GORM_SEQUENCE" +
+						` OPTIONS (sequence_kind = "bit_reversed_positive")`).Error; err != nil {
+						return err
+					}
+					f.HasDefaultValue = true
```

Comment:
Based on the above `if` statement, this should already be true, and thus redundant.